assets/css/04-objects/_buttons.scss

```diff
@@ -1,47 +1,47 @@
-/**
- * Button classes
- */
-
-.btn {
-	@include btn;
-}
-
+/**
+ * Button classes
+ */
+
+.btn {
+	//@include btn;

```

Comment:
My bad. Die had ik uit een eerder project uitgezet.
Lege classes is idd niet nodig. Wel zou ik standaard alles uitzetten. Zo kun je per site eenvoudig aanzetten welke je echt nodig hebt en houdt je de initiële css klein.